scripts/maintenance/computational-clusters/osparc_clusters.py

```diff
@@ -336,6 +379,10 @@ def _color_encode_with_state(string: str, ec2_instance: Instance) -> str:
     )
 
 
+def _color_encode_with_threshold(string: str, value, threshold) -> str:
+    return string if value > threshold else f"[red]{string}[/red]"
```

Comment:
TIP:   small helper for coloring
```
DANGER = f"[red]{}[/red]"
DANGER.format("Careful pedro is around!")

```
For nice UX with colors I recommend https://getbootstrap.com/docs/4.0/components/alerts/ :-)